geode-core/src/main/java/org/apache/geode/management/internal/configuration/persisters/ConfigurationPersister.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.geode.management.internal.configuration.persisters;
+
+import org.apache.geode.annotations.Experimental;
+import org.apache.geode.cache.configuration.CacheConfig;
+import org.apache.geode.management.internal.configuration.domain.ClusterConfigElement;
+
+/**
+ * Defines the behavior to persist a configuration change on a locator
+ * {@link org.apache.geode.distributed.ConfigurationPersistenceService}. Created with an object of
+ * type ClusterConfigElement, which represents the configuration change.
+ */
+@Experimental
+public abstract class ConfigurationPersister<T extends ClusterConfigElement> {
+  T config;
```

Comment:
`private` rather `package private`?

Reply:
Unfortunately because the LocatorClusterManagementService in a different package needs to access this, it can't be package private.